specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json

```diff
@@ -226,9 +226,65 @@
         "daily",
         "hourly",
         "minutely",
-        "secondly"
+        "secondly",
+        "microsecond"
       ]
     },
+    "ImputeMode": {
+      "type": "string",
+      "description": "can be one of auto, previous, linear, fixed, zero or notFill.",
```

Comment:
nit, capital "C"

Reply:
Updated.

---
In reply to: [560686437](https://github.com/Azure/azure-rest-api-specs/pull/12487#discussion_r560686437) [](ancestors = 560686437)